src/components/scrollToTop/index.js

```diff
@@ -10,8 +10,8 @@ class ScrollToTop extends Component {
 
   componentDidUpdate(prevProps) {
     // compare pathname, which ignoring changes in URL hash
-    const currentPath = this.props.location && this.props.location.pathname;
-    const previousPath = prevProps.location && prevProps.location.pathname;
+    const currentPath = this.props.location && this.props.location.pathname && this.props.location.search;
+    const previousPath = prevProps.location && prevProps.location.pathname && prevProps.location.search;
```

Comment:
I think this should be concatenating `pathname` and `search' right? Not just doing a boolean operation to pick one of them?